be/src/service/internal_service.cpp

```diff
@@ -411,13 +411,6 @@ Status PInternalServiceImplBase<T>::_exec_plan_fragment(brpc::Controller* cntl,
     if (is_pipeline) {
         return _exec_plan_fragment_by_pipeline(t_request, t_request);
     } else {
```

Comment:
is there compatibility issue during gray-scale udpate？ it seems that non-pipeline engine code should not removed in current version,  instead, it can be removed safely in next version.